app/src/main/res/drawable/ic_oppia_icon_mint.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="164.37"
```

Comment:
Is 24dp the right size? Ditto below for other imported SVGs.

Reply:
I have removed unwanted icons.  Chantel has provided splash page with logo. That works fine.